kumascript/macros/ListSubpagesForSidebar.ejs

```diff
@@ -34,111 +30,90 @@ const overview = mdn.localString({
     'ko': '개요',
     'ru': 'Обзор',
     'zh-CN': '概述',
+    'zh-TW': '概述'
 });
 
-var parent = await wiki.getPage(path);
-var pages = includeParent ? [{
-    ...parent,
-    title: overview
-}, ...parent.subpages] : [...parent.subpages];
-var containsTag = page.hasTag;
-var htmlEscape = kuma.htmlEscape;
-
-var output = '';
-var openTag = '<ol>';
-var closeTag = '</ol>';
-var code    = !$1 ? '<code>'  : '';
-var endcode = !$1 ? '</code>' : '';
-
-// Strings
-var commonl10n = web.getJSONData('L10n-Common');
-var text = {
-    'translate': mdn.getLocalString(commonl10n, '[Translate]'),
-    'title': mdn.getLocalString(commonl10n, 'TranslationCTA'),
-};
-
-var badges = {
-  ExperimentalBadge : await template("ExperimentalBadge"),
-  NonStandardBadge : await template("NonStandardBadge"),
-  DeprecatedBadge : await template("DeprecatedBadge"),
-  ObsoleteBadge : await template("ObsoleteBadge"),
+const parent = await wiki.getPage(path);
+const pages = parent.subpages;
+
+// trim english title for sorting
+for (const page of pages) {
+    page.title = trimTitle(page.title);
+}
+pages.sort((a, b) => a.title.localeCompare(b.title)) // sorted by english title for subpages    
+
+// always keep the parent page on top, so insert it after sorting
+if (includeParent && parent.url) {
+    pages.unshift({
+        ...parent,
+        title: overview
+    });
+}
+
+let output = '';
+
+const containsTag = page.hasTag;
+const htmlEscape = kuma.htmlEscape;
+const openTag = '<ol>';
+const closeTag = '</ol>';
+const code = !$1 ? '<code>' : '';
+const endcode = !$1 ? '</code>' : '';
```

Comment:
Can we extract a variable `const wrapInCode = !$1;` above where the other `$n` parameters are defined?